package.json

```diff
@@ -39,34 +39,47 @@
     "lint:report": "eslint . --output-file ./eslint_report.json --format json  --ext .ts,.js,.tsx | true"
   },
   "dependencies": {
-    "react-native-modal": "11.5.6"
+    "react-native-modal": "13.0.1"
   },
   "peerDependencies": {
-    "react": ">=16.8.3",
-    "react-native": ">=0.59.0"
+    "react": "*",
+    "react-native": ">=0.65.0"
```

Comment:
So, this works for current react native version in pos and upcoming version change?

Reply:
Yes, Tested on my local